modules/clientmanagements/client/controllers/list-clientinventory.client.controller.js

```diff
@@ -25,12 +27,67 @@
     $scope.$state = $state;
     $scope.authentication = Authentication;
 
-    $scope.cancelButton = function() {
+    $scope.cancelButton = function () {
       if ("admin" === Authentication.user.roles[0]) {
         $state.go('mainmenuadmin');
       } else {
         $state.go('mainmenu');
       }
     };
+
+    $scope.moveToClient = function () {
```

Comment:
Same with this, this is all stuff from my branch and will be merged with my pull request, it shouldn't be here.